### PR TITLE
Fix app going back to home screen after scanning BP passport

### PR DESCRIPTION
**Pull request by msasikanth**
https://app.clubhouse.io/simpledotorg/story/2063/app-shows-home-screen-after-scanning-a-bp-passport

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/home/HomeScreenEvent.kt [hunk 1]
```diff
@@ -5,6 +5,7 @@ import org.simple.clinic.patient.Patient
 import org.simple.clinic.patient.businessid.Identifier
```
Comment (Honey14) [inline app/src/main/java/org/simple/clinic/home/HomeScreenEvent.kt:4]: When we don't find the patient we open the search screen. Can we update the commit message to reflect that?

**Review by Honey14 — CHANGES_REQUESTED** [review]
Looks good. Left one comment. PTAL

File: app/src/main/java/org/simple/clinic/home/HomeScreenEvent.kt (hunk 1 above)
Comment (msasikanth) [reply app/src/main/java/org/simple/clinic/home/HomeScreenEvent.kt:4]: Oh wait, I thought I pushed that change. My bad will fix it 😅

**Review by Honey14 — APPROVED** [review]
Looks good

**Pull request merged**